contracts/eden/src/inductions.cpp

```diff
@@ -15,12 +15,35 @@ namespace eden
          row.id = id;
          row.inviter = inviter;
          row.invitee = invitee;
-         row.witnesses = witnesses;
-         row.endorsements = {};
+         row.endorsements = witnesses.size() + 1;
```

Comment:
`update_profile` also needs to be updated.

Reply:
nice catch! working on it!

Reply:
done!